### ISSUE TITLE
vPrev->vCurrent nmdc:Database migration script on each release if needed

### ISSUE THREAD

**Issue by dwinston** [microbiomedata/nmdc-schema#1019]
given that a principal use case of our schema-as-software (the importable `nmdc_schema` package) is to support the maintenance of schema-conformant databases (e.g. `nmdc:Database` JSON), it seems reasonable for each schema-as-software release to include migration code from the previous release if appropriate.

@turbomam what do you think?

This even seems like something that could be supported at the `linkml` level...

**Comment by turbomam** [comment]
I agree. There are some pre-alpha LInkML tools for this kind of thing, but I think we should assume it will be our responsibility to write in either a specific or general way for the time being.

**Comment by turbomam** [comment]
https://github.com/linkml/linkml-transformer

**Comment by eecavanna** [comment]
We have been doing something like this for the past few non-patch releases. See: https://github.com/microbiomedata/nmdc-schema/tree/main/nmdc_schema/migrators

**Comment by turbomam** [comment]
Closing: per-release migrators are implemented in `nmdc_schema/migrators/` and have shipped for many releases.
